packages/checkout-ui-extensions/docs/staticPages/examples/.eslintrc.js

```diff
@@ -0,0 +1,8 @@
+// See the root .eslintrc for more overrides.
```

Comment:
Hmm, do we need to copy anything? https://github.com/Shopify/checkout-web/blob/master/.eslintrc.js#L113-L127